data/dashboard.yml

```diff
@@ -4,6 +4,14 @@
     - name: Publishing apps
     - name: APIs
     - name: Supporting apps
+      sites:
```

Comment:
I'm not sure this is an application like the others (it isn't a web thing at least). Perhaps a new section on this dashboard for transition-y stuff like https://github.com/search?q=topic%3Agovuk-transition+org%3Aalphagov?

Reply:
Good point.  I've retooled the commit.